roles/openshift_health_checker/openshift_checks/ram_diskspace_availability.py

```diff
@@ -0,0 +1,42 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+from openshift_checks.mixins import NotContainerizedMixin
+
+
+class RamDiskspaceAvailability(NotContainerizedMixin, OpenShiftCheck):
```

Comment:
Should the memory and diskspace checks be separated?

I would think that Memory checks would apply equally to containerized/non-containerized.

Are there plans to also add a disk space check for containerized installs as well?